user/ball.c

```diff
@@ -54,8 +53,11 @@ int main(const char *argv[], int argc)
 		draw_flush();
 
 		syscall_process_sleep(75);
+		syscall_object_read_nonblock(KNO_STDIN, (void *) &stop, 1);
```

Comment:
Usually a cast is a sign of some underlying trouble.  What's the reason for it here?